packages/monitor-v2/src/monitor-og/README.md

```diff
@@ -62,6 +62,8 @@ All the configuration should be provided with following environment variables:
   `MNEMONIC` for signing execution transactions.
 - `SUPPORTED_BONDS` is a mapping of supported bond tokens and bond values. This is required when running in automated
   support mode. Only oSnap modules with exact match of bond token and bond value will be supported.
+- `SERVER_ERROR_DISPUTE_PERIOD` is the number of seconds left before challenge window ends when proposals can be
```

Comment:
This is a really interesting concept. +1

If we're this late in a liveness period, wouldn't we assume that the bot has _already_ checked it many times, given the only livenesses we check are long?

Do you think it's safe to assume that if these have been down/erroring for _days_ that the operator would have received the error from the bot (via page or other notification) and manually disputed? (Note: presumably the error logs give the operator enough info to construct a manual dispute)

Reply:
This is good point and really quite unlikely IPFS gateway or GraphQL server being unreachable all the time since posting proposal on-chain. Maybe better alternative is for the dispute bot to send error when it cannot verify due to network error, but not dispute?